main.js

```diff
@@ -158,6 +158,10 @@ define(function (require, exports, module) {
             type: "bramble:loaded"
         }), "*");
 
+        //Below are methods to change the preferences of breackets, more available at:
+        // https://github.com/adobe/brackets/wiki/How-to-Use-Brackets#list-of-supported-preferences
+        PreferencesManager.set("insertHintOnTab", true );
```

Comment:
Remove the extra space between `true` and the `)`.  Looks good otherwise.

You've tested in our Brackets and it works?